### PR TITLE
Update homepage for post-2020 show

### PR DESCRIPTION
**Pull request by dikunong**
* Update homepage after 2020 show
* Change CSS styling to fit best practices

## New CSS style

I've been meaning to restyle our CSS for ages ([as referenced in a PR review from Alex K way back](https://github.com/UoAEngineeringRevue/engrevue-website/pull/1#discussion_r382993509)), finally got around to it. TL;DR is:
* CSS classes are named with `skeleton-case`, not `camelCase` (they are still referenced in `camelCase` in the JS though, because JS, lol)
* Order CSS declarations alphabetically within classes

This makes our CSS in line with industry standards and also generally a bit easier to read. We should stick to it going forward :)

### CODE REVIEW

**Comment by dikunong** [comment]
@alex-ruddell I'd like to draw your attention to the commit `d5ccef7`: *"Fix bug with media query shrinking body text with too high specificity"*. This is a good opportunity to learn about how Gatsby handles CSS.

What happened was that a media query in `sponsor.module.css` shrunk the size of the generic body text element, `<p>`. This is fine in isolation (read: local testing), and it worked as intended on the Sponsorship page. But - Gatsby is a single-page application framework, so when it compiles the CSS it crushes it all into a single file. This means that CSS written in specific pages/components is actually present in other parts of the website.

Normally, nothing happens as a result - Gatsby recognises that CSS classes with the same name on a specific page are intended for that page, so when it compiles the CSS it appends a unique ID so it doesn't spill like that. The problem here was that you didn't use a class, but a universal HTML element selector. HTML elements like `<p>`, `<h1>`, `` etc are generic and used everywhere. Since Gatsby compiles the CSS into a single file, when you style a generic element, the style then applies to everywhere it occurs - which means on other pages, too!

So, the media query was shrinking `<p>` text on every page instead of just the Sponsorship page. This is undesired and so I've resolved it by just deleting the CSS class. Generally, it's always best to use classes to select styling unless you have a good reason to be more or less specific. If your reason is that you want to style a generic HTML element, we have these styles in `layout.css` for consistency across the website. If changes are desired, they should be done here and tested across all pages!  
(Looking at the page again now, the headers need to be bigger rather than the text smaller. Too small text is just not fun to read and also inaccessible.)

TL;DR **Use CSS class selectors instead of element selectors unless you have a reason to style everything of a kind. If you style everything of a kind, do it in the right place and test it everywhere it happens.** Don't feel bad about it - everything's a learning opportunity :)

(also an example of why good code review is important - because I totally missed this last time!)

**Review by alex-ruddell — APPROVED** [review]
**Looks good to me man!** I like the styling of the code and all that - definitely something I will employ moving forward to keep it to standard.

- Didn't spot any typos or any major issues, looks pretty standard! Had a play around with it too and it is looking nice.
- Only thing I noticed, which is probably personal preference; the "Like us on Facebook" button has quite a lot of padding. It is equal to that of the "WATCH NOW" button in the top corner, but that one looks fine because it is part of the header. If it doesn't look off to you, though, don't bother changing it :)

_Not something you've specifically changed in this PR, but this had made me take a closer look at the blurb on the index page under "About Us"! I think we should cut it down a bit and have it a bit more snappy - our show also isn't technically put on by the Faculty of Engineering, so there's some stuff we can change in there aswell! Will talk about this with you at some stage I am sure.__

**Comment by dikunong** [comment]
Was just thinking about that today haha, the blurb is a real wall of text. We can rewrite it for sure!
